egs/wsj/s5/steps/nnet3/lstm/make_configs.py

```diff
@@ -305,7 +331,7 @@ def ParseSpliceString(splice_indexes):
     if (num_hidden_layers < args.num_lstm_layers):
         sys.exit("--num-lstm-layers : number of lstm layers has to be greater than number of layers, decided based on splice-indexes")
 
-    left_context = left_context + args.bptt_truncation_width + args.context_sensitive_chunk_width
+    left_context = left_context + args.chunk_left_context
```

Comment:
This doesn't have to delay this merge, but I don't like the fact that chunk_left_context is provided to this script, because it doesn't have to do with the model per se.  The left-context and right-context of the model should be, IMO, just the intrinsic left and right context, and we can then add the chunk_left_context just before we pass the arg to get_egs.sh.  This will be more flexible in case we decide to do things like initially train on a certain (chunk-size, chunk-left-context) and then later in training, increase those values.